src/text-field.jsx

```diff
@@ -55,6 +55,7 @@ const TextField = React.createClass({
     rowsMax: React.PropTypes.number,
     style: React.PropTypes.object,
     type: React.PropTypes.string,
+    underline: React.PropTypes.bool,
```

Comment:
Let me know which one you want! I'm fine with either :smile: 

Now that I think about it, as a general rule, using a "positive-form" adjective might have a slight advantage over a verb/noun because some people specify boolean props as truthy without explicitly specifying the value of the prop as true.

For example some people and use eslint rules for this pattern:

```jsx

/* CORRECT */
<Component
   active
/>

/* WRONG */
<MyComponent
   active={true}
/>

/* CORRECT */
<MyComponent
   active={false}
/>
```